pom.xml

```diff
@@ -6,51 +6,55 @@
 
   <groupId>kafka.vertx</groupId>
   <artifactId>demo</artifactId>
-  <version>0.0.1</version>
+  <version>0.0.2-SNAPSHOT</version>
 
   <properties>
     <kafka.version>2.3.0</kafka.version>
-    <vertx.version>3.8.4</vertx.version>
+    <vertx.version>4.0.0-SNAPSHOT</vertx.version>
```

Comment:
This version of the vertx-dependencies doesn't seem to be in Maven central. Is there a snapshot repo we need to add to the pom.xml

Reply:
We're using https://central.sonatype.org/pages/ossrh-guide.html to deploy snapshots, so you should add https://oss.sonatype.org/content/repositories/snapshots/ (preferrably in your `~/.m2/settings.xml`)